cachito/workers/scm.py

```diff
@@ -126,7 +162,11 @@ def fetch_source(self):
         archive_path = self.sources_dir.archive_path
         if archive_path.exists() and tarfile.is_tarfile(archive_path):
```

Comment:
Should we remove the `is_tarfile` call from here since it's done during `_verify_archive`?

Reply:
Done